<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Event</class>
 <widget class="QWidget" name="Event">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>181</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Создать событие</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Дата начала</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="startDateEdit">
      <property name="dateTime">
       <datetime>
        <hour>15</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2024</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>14</hour>
        <minute>0</minute>
        <second>59</second>
        <year>2100</year>
        <month>12</month>
        <day>30</day>
       </datetime>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="startTimeEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Дата конца</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="endDateEdit">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2025</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="minimumDate">
       <date>
        <year>2025</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="endTimeEdit">
      <property name="maximumDateTime">
       <datetime>
        <hour>18</hour>
        <minute>59</minute>
        <second>59</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="minimumTime">
       <time>
        <hour>0</hour>
        <minute>1</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="create_event">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>530</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>381</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Название события</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Приоритет</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="priorityCombo">
      <item>
       <property name="text">
        <string>Менее важно</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Средне важно</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Важно</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Тип</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="typeCombo">
      <item>
       <property name="text">
        <string>Другое</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Встреча</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Конференция</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Презентация</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Описание</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="descEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Участники</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="parts"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="result">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>540</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
